README.md

```diff
@@ -53,14 +58,15 @@ Navigate into the extension directory and initialize it as a Git repository:
 
 ```bash
 cd saltext-apache
-git init --initial-branch=main
+git init 
 touch requirements/{changelog,dev,docs,docs-auto,lint,tests}.txt
 pip install pre-commit # may not need to be done again if in a pre-existing virtual environment
 pre-commit install
 git add .
 git commit -m "Initial commit of extension framework"
 ```
 > *You may need to `pip install` `git-filter-repo` and `pylint`.
+> If you are not using git < 2.28, it will default to the branch being "master" instead of "main" - you won't be able to change it until after you do a commit.
```

Comment:
**issue (blocking):** it's my understanding that the main problem with version below that mark is the lack of support for the `initial-branch` option provided in the commands above

```suggestion
> If you are using git < 2.28, the `initial-branch` option is not present and `git init` will default to the branch being "master" instead of "main". Note that you won't be able to change it with `git branch -m master main` until after your first commit.
```